glide.yaml

```diff
@@ -1,6 +1,6 @@
 package: github.com/uber/ringpop-go
 import:
-- package: github.com/Sirupsen/logrus
+- package: github.com/sirupsen/logrus
```

Comment:
I think you should also need to update `github.com/uber-common/bark` in glide.lock